game/hud/src/services/session/layout.ts

```diff
@@ -109,18 +121,80 @@ const initialState = {
   widgets: defaultWidgets(),
 }
 
-let getInitialState = () => {
-  var storedState = JSON.parse(localStorage.getItem(localStorageKey)) as LayoutState;
+interface Size {
+  width: number;
+  height: number;
+}
+
+const px2pcnt = (pos: Position, screen: Size) : Position => {
+  // converts pixel co-ordinates into % of display size
+  return {
+    x: pos.x / screen.width,
+    y: pos.y / screen.height,
+    width: pos.width,           // don't scale width/height
+    height: pos.height,
+    scale: 1
+  };
+};
+
+const pcnt2px = (pos: Position, screen: Size) : Position => {
+  // converts % co-ordinates into pixels for current display
+  return {
+    x: screen.width * pos.x,
+    y: screen.height * pos.y,
+    width: pos.width,           // don't scale width/height
+    height: pos.height,
+    scale: 1
+  }
+};
+
+const forceOnScreen = (pos: Position, screen: Size) : Position => {
```

Comment:
Doesn't 8.1 https://github.com/airbnb/javascript#arrow-functions mean this is ok.  That is, this is an arrow function not a function expression (treated separately in the guide)

See 8.5 for the following example:

```JavaScript
// good
const itemHeight = (item) => {
  const { height, largeSize, smallSize } = item;
  return height > 256 ? largeSize : smallSize;
};
```